index.js

```diff
@@ -0,0 +1,258 @@
+const userDb = require('./data/helpers/userDb.js');
+const postDb = require('./data/helpers/postDb.js');
+const express = require('express');
+const cors = require('cors');
+
+const server = express();
+server.use(cors());
+server.use(express.json());
+
+const toCaps = (request, response, next) => {
+    request.name = request.body.name.toUpperCase();
```

Comment:
Did you mean to create a new property on the `req` object? Is there an advantage or disadvantage to adding a name property to the req versus resetting the req body's name property to a newly modified value?